tools/fastlane-plugin/lib/fastlane/plugin/bugsnag/actions/upload_symbols_to_bugsnag.rb

```diff
@@ -6,21 +6,23 @@ class UploadSymbolsToBugsnagAction < Action
         File.join(File.dirname(__FILE__), '..', '..', '..', '..', '..', 'bugsnag-dsym-upload'))
 
       def self.run(params)
-        options = {}
-        options = options_from_info_plist(params[:config_file]) if params[:config_file]
-
-        if (params[:config_file] == default_info_plist_path || options[:apiKey].nil? || options[:apiKey].empty?)
-          # Load custom API key if config file has not been overridden or the API key couldn't be found in config files
-          options[:apiKey] = params[:api_key] unless params[:api_key].nil?
-        else
-          # Print which file is populating version and API key information since the value has been
-          # overridden
-          UI.message("Loading API key from #{params[:config_file]}")
+        # If we have not explicitly set an API key through env, or parameter
+        # input in Fastfile, find an API key in the Info.plist in config_file param
+        if params[:config_file] && params[:api_key] == nil
```

Comment:
I don't like mutating params here, we should determine an apikey as a local variable to pass into the later upload_args method.

Reply:
Done: https://github.com/bugsnag/bugsnag-dsym-upload/pull/42/commits/48765f86c729f812fb8a4a9dff146887f68165b0

Reply:
`api_key` local is fed in to the upload args in this commit: https://github.com/bugsnag/bugsnag-dsym-upload/pull/42/commits/a910a01c18ea50c067d3f42fa459484deb5edea0